ui/src/assets/images/guardian-logo-320.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="320" height="60">
```

Comment:
There is an SVGO plugin for that (disabled by default): https://github.com/svg/svgo/blob/master/plugins/removeXMLNS.js